### ISSUE TITLE
Minecraft worlds won't load with Dense Ores 1.6.2 and Thaumcraft

### ISSUE THREAD

**Issue by AlexPlus117** [rwtema/DenseOres#92]
I can't create or load worlds with latest versions of Dense Ores and Thaumcraft. Here is the log-file: http://pastebin.com/eBz3CCWY

**Comment by Nincodedo** [comment]
Would it be possible to get the full crash log? This doesn't really tell the whole story.

**Comment by AlexPlus117** [comment]
It's a really long log-file, so I can't upload the full log to pastebin, but I've uploaded some of it.http://pastebin.com/W5yfdn8y

**Comment by AlexPlus117** [comment]
Any idea?

**Comment by UndeadZeratul** [comment]
Given a cursory glance, I noticed a call within Thaumcraft to load EMC values (I assume through EE3).  Since you have Project E, would that be causing the issue?  An outdated version of Project E?  If not, then maybe this is something Project E needs to account for?

If you remove either Thaumcraft or Project E, does your pack work?  What if you had a pack with only those two mods, would it crash?

Also, what does your dense ore config look like?  Maybe there's an issue with how you have it set up?

**Comment by AlexPlus117** [comment]
If I delete either ProjectE or Thaumcraft, the worlds I create load correctly. I put dense ores config here on pastebin: http://pastebin.com/SSLMXvRr

**Comment by UndeadZeratul** [comment]
Hmm, so that tells me that there's an interaction between Project E and Thaumcraft going awry here.  If you remove Dense Ores, are you able to create worlds?  Maybe this is just something that happens when all 3 mods interact?

I noticed you have dense amber and dense cinnabar ores being defined, maybe those are causing issues?  If nothing else, try removing those dense ore definitions and see if that helps.

**Comment by AlexPlus117** [comment]
Yes, I just removed dense cinnabar and dense amber ore and the worlds load with no problems, but this means that there's an error on those ore generation.

**Comment by UndeadZeratul** [comment]
Not entirely, it's not purely Thaumcraft + Dense Ores, nor is it purely Thaumcraft + Project E.  It's all 3 of these mods interacting together.

Now, I don't know who is really at fault here, although I'm leaning more towards Project E or Thaumcraft, as something in their EMC code is causing this mod's ores to break.  My guess based on just glancing at the mods in question, it seems there's an infinite loop going on with the dense ores and their regular ore counterparts.  You don't necessarily have to close this issue here, as Tema can make that decision, but I'd at least let the other two mods know of the issue with this interaction.
